utils/motorcontroller/SparkMAXFactory.java

```diff
@@ -54,6 +61,19 @@ public static SparkMAXLite buildFactorySparkMAX(int deviceID, MotorConfig.Motor_
         return buildSparkMAX(deviceID, defaultConfig, motorType);
     }
 
+    public static SparkMAXLite buildFactorySparkMAX(int deviceID, MotorConfig.Motor_Type motorType, boolean velocityImportant, boolean positionImportant) {
```

Comment:
two booleans is a little hokey. Probably cleaner to have a single parameter that defines what kind of control we want

Reply:
I changed it so that the user just passes in the config they want and left more specific comments on the purpose of each config. Does that sound fine? An enum felt really repetitive.